src/pages/trade-types/options/digital-options/up-and-down/index.tsx

```diff
@@ -1,9 +1,22 @@
 import React from 'react'
+import { Helmet } from 'react-helmet'
 import { WithIntl } from 'components/localization'
 import UpAndDownDigitalOptionsPage from 'features/pages/options/digital/up-down'
+import { isBrowser } from 'common/utility'
 
 const DigitalOptions = () => {
-    return <UpAndDownDigitalOptionsPage />
+    const canonicalUrl = isBrowser()
+        ? `${window.location.origin}/trade-types/options/digital-options/up-and-down`
```

Comment:
Hi @mitra-deriv are we doing this just UP and Down?

Reply:
Hi yes @fasihali-deriv cause the old option page gonna redirect to this path only and this is the default tab

Reply:
Also one thing can we use `getDomainUrl()`. 
I am not sure just asking 😬  🙏

Reply:
@fasihali-deriv The `getDomainUrl()` only get the hostname , like deriv.com with out https:// , I think we can go ahead with current implementation , thanks  🙏